### DISCUSSION TITLE
Backup Notifications

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#484] [Ideas] [answered]
Notifications about successfully completed backups are optional or alternatively you can choose if you want to be notified only in case of errors (if that makes sense).

**Comment by szaimen** [comment]
In my opinion a notificatioin is mandatory because it could happen that the backup failed and didn't send the notification (e.g. because of a bug) but you would think the backup was successful in that case. So better require to send a notification every day to be save in that regard.

**Reply by pmorch** [reply]
Lets take this a step further. For me, nextcloud is _one_ of the services I manage. I get emails when miscellaneous cron jobs on my servers fail. Imagine if I got emails for every run of every cronjob on all servers, regardless of whether they succeeded or failed. That is noise and not sustainable.

Please reconsider this.

**Comment by szaimen** [comment, marked as answer]
Will be possible with https://github.com/nextcloud/all-in-one/pull/3516

**Reply by szaimen** [reply]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
